tests/README.md

```diff
@@ -1,99 +1,39 @@
 # cass-operator integration tests
 
-[Ginkgo](https://onsi.github.io/ginkgo/) is used alongside kubectl
-for integration testing of the operator.
+[Ginkgo](https://onsi.github.io/ginkgo/) is used alongside kubectl for integration testing of the operator.
 
 ## Prerequisites
-Install mage using the instructions
-[here](https://github.com/magefile/mage#installation)
+Install kubectl using the instructions [here](https://kubernetes.io/docs/tasks/tools/install-kubectl) 
 
-Install kubectl using the instructions
-[here](https://kubernetes.io/docs/tasks/tools/install-kubectl) 
+The tests use Kustomize to deploy the cass-operator. Kustomize is installed if not present in the machine. We test against 4.1.x series of Kustomize at this point, but newer 3.x releases should work also. 
 
-## Running the tests
-The tests themselves expect a running k8s cluster, with at least 6 worker nodes,
-and kubectl to be configured to point at the cluster.
-
-You can either stand up your own cluster and point kubectl to it yourself, or use
-a mage target to bootstrap and configure a cluster for you.
-
-Refer to the [k8s targets](../docs/developer/k8s_targets.md) for more information on
-supported k8s flavors.
-
-### Using preconfigured targets for cluster management
-
-To let a mage bootstrap a new cluster, run all of the tests,
-and then tear down the cluster at the end, run the `k8s:runIntegTests` target
-with the desired k8s flavor specified:
-
-#### Using a k3d cluster
-```
-M_K8S_FLAVOR=k3d mage k8s:runIntegTests
-```
-
-#### Using a KIND cluster
-```
-M_K8S_FLAVOR=kind mage k8s:runIntegTests
-```
-
-If you just want to have the cluster stood up and configured for you, but
-not automatically run the tests or tear down, run:
-
-#### Using a k3d cluster
-```
-M_K8S_FLAVOR=k3d mage k8s:setupEmptyCluster
-```
+You will also need a running Kubernetes cluster. We use [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation), but others should work also. Note that the storageClass is set to ``default`` in the tests.
```

Comment:
minor nit - I would change `storageClass` to either `StorageClass` or `storage class`.